packages/server-web/src/lib/tracked-time-extractor.js

```diff
@@ -0,0 +1,20 @@
+module.exports = { extractTrackedTimes };
+
+function extractTrackedTimes(inputString) {
+  const regex = /#TRACK\((\{.*?\:.*?\})\)/gi;
```

Comment:
It should probably match only everything between the `(...)` and not check for JSON-like stuff

Reply:
Not sure... should it also match `#TRACK({ i_am_invalid_json })`?

Reply:
It probably will but `JSON.parse` should throw in that case anyways. I wouldn't want to try to match JSON with a regexp :)